Dockerfile

```diff
@@ -1,24 +1,22 @@
 FROM node:18.16.0-alpine@sha256:44aaf1ccc80eaed6572a0f2ef7d6b5a2982d54481e4255480041ac92221e2f11 AS builder
 WORKDIR /app
-RUN [ "yarn", "set", "version", "1.22.17" ]
 COPY /src ./src
 COPY package.json ./
-COPY yarn.lock ./
+COPY package-lock.json ./
 
-RUN yarn install
-RUN yarn build
+RUN npm install
+RUN npm run build
 
-# 'yarn install --production' does not prune test packages which are necessary
+# 'npm install --omit=dev' does not prune test packages which are necessary
```

Comment:
Sorry, I meant that this command _does_ prune dev dependencies - so you don't need to delete `node_modules` explicitly.